packages/flutter_tools/test/integration.shard/exit_code_test.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.8
+
+import 'package:file/file.dart';
+import 'package:flutter_tools/src/base/io.dart';
+
+import '../src/common.dart';
+import 'test_utils.dart';
+
+void main() {
+  final String dartBin = fileSystem.path.join(getFlutterRoot(), 'bin', 'dart');
+  Directory tempDir;
+
+  setUp(() {
+    tempDir = createResolvedTempDirectorySync('exit_code_test.');
```

Comment:
Does this need to be `flutter_exit_code_test.` or is the `flutter_` added by `createResolvedTempDirectorySync`?

Reply:
yup - https://github.com/flutter/flutter/blob/08a70e7ac836245cfb6e6156835fca2439722899/packages/flutter_tools/test/integration.shard/test_utils.dart#L32